src/datastore/postgres/PgDatastore.ts

```diff
@@ -209,7 +209,7 @@ export class PgDatastore implements Datastore {
             status: entry.status,
             user_id: entry.user_id,
         };
-        const statement = PgDatastore.BuildUpsertStatement("teams", "id", props);
+        const statement = PgDatastore.BuildUpsertStatement("teams", ["id"], [props]);
         await this.postgresDb.none(statement, props);
```

Comment:
Does this still need to be called with `props` or are the values inserted in `pgp.helpers.insert(values, cs)`?

Reply:
In your code, `props` is the actual data, so i'm not sure about your question, whether we can skip the actual data to be inserted?

Reply:
> why `conflictKeys` and `values` became arrays

Because they were changed to be generic, allowing the last parameter of `BuildUpsertStatement` to be an array objects, which will automatically generate a multi-row insert.

Reply:
Looks redundant to me. If you don't remove it, I'll remove the 2nd argument to `.none` in a follow-up PR.
[insert](https://vitaly-t.github.io/pg-promise/helpers.html#.insert) does put the data in the statement while not generating placeholders.